airflow/operators/python.py

```diff
@@ -644,10 +646,13 @@ class PythonVirtualenvOperator(_BasePythonVirtualenvOperator):
         exit code will be treated as a failure.
     :param index_urls: an optional list of index urls to load Python packages from.
         If not provided the system pip conf will be used to source packages from.
+        If set to ``[]``, ``--no-index`` will be used to ignore package index.
```

Comment:
It's hard to figure out the fact that if `index_urls=[]`(empty list), it converted to `--no-index` pip option.
I added comment to help users discover this more quickly.